src/Mod/Spreadsheet/App/Cell.cpp

```diff
@@ -72,10 +72,8 @@ const int Cell::ALIGNMENT_VERTICAL   = 0xf0;
 
 /**
   * Construct a CellContent object.
-  *
-  * @param _row   The row of the cell in the spreadsheet that contains is.
-  * @param _col   The column of the cell in the spreadsheet that contains is.
-  * @param _owner The spreadsheet that owns this cell.
+  * @param _address  The adress of the caell
```

Comment:
They have been removed from the method(or where never there). Not by me. The comments are just reflecting what's existing to shut-up doxygen.